apps/src/code-studio/pd/components/customSchoolInfo.jsx

```diff
@@ -0,0 +1,121 @@
+import React, {PropTypes} from 'react';
+import {FormGroup, ControlLabel, HelpBlock} from 'react-bootstrap';
+import Select from "react-select";
+import {ButtonList} from '../form_components/ButtonList.jsx';
+import FieldGroup from '../form_components/FieldGroup';
+import {STATES} from '../../../geographyConstants';
+
+const VALIDATION_STATE_ERROR = "error";
+
+const SCHOOL_TYPES = {
+  PUBLIC: "Public school",
+  PRIVATE: "Private school",
+  CHARTER: "Charter school",
+  OTHER: "Other"
+};
+
+export default class CustomSchoolInfo extends React.Component {
+  static propTypes = {
+    school_info: PropTypes.object,
+    onSchoolInfoChange: PropTypes.func,
+    errors: PropTypes.object
+  };
+
+  updateOnSchoolInfoChange = (school_info) => {
+    if (this.props.onSchoolInfoChange) {
+      this.props.onSchoolInfoChange(school_info);
+    }
+  };
```

Comment:
should I just remove this method and call `onSchoolInfoChange` directly? (and make it required)?

Reply:
That sounds good 👍